docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -102,11 +105,13 @@ A JSON object that defines the rule's values:
 * <<req-fields-ml>>
 * <<req-fields-threat-match>>
 * <<req-fields-new-terms>>
+* <<req-fields-esql>>
 * <<opt-fields-all>>
 * <<opt-fields-query-eql>>
 * <<opt-fields-threat-match>>
 * <<opt-fields-query-threshold>>
 * <<opt-fields-eql-query-threshold>>
+* <<opt-fields-eql-query-threshold-new_terms-esql>>
 
 [[ref-fields-all]]
 ===== Required fields for all rule types
```

Comment:
Oh yeah, scratch my comment about the index field. I'm so used to defining indices for the other rule types that I forget the data source is defined on the query level when creating/updating ES|QL rules. 

That said, do we still need to add the note about required metadata fields to [this spot](https://security-docs_4052.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#req-fields-query-threshold)? More specifically, do we need to tell users that they need to include the metadata operator if they're writing a non-aggregating ES|QL query?

Reply:
I don't think so. This should and will be covered in a separate section. Maybe give link to that section from here?
Otherwise, we will duplicate this info and it would be harder to maintain in future, if we change something

Reply:
Yeah, we can add a line telling users to look at the [ES|QL rule docs](https://www.elastic.co/guide/en/security/master/rules-ui-create.html#create-esql-rule) for more info on writing ES|QL queries.

Reply:
Resolved in [576067e](https://github.com/elastic/security-docs/pull/4052/commits/576067e0d7eee9eb46c4af9eb17e0c4e9e74f6f8).